redhat-director-scripts/docker/trilio-datamover/Dockerfile_rhosp16.2

```diff
@@ -16,10 +16,13 @@ USER root
 ADD trilio.repo /etc/yum.repos.d/
 
 RUN dnf clean all
-RUN dnf config-manager --enable openstack-16-for-rhel-8-x86_64-rpms --enable rhel-8-for-x86_64-baseos-rpms
-RUN dnf install python3-tvault-contego puppet-triliovault python3-s3fuse-plugin -y
-RUN yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical
-RUN dnf config-manager --disable openstack-16-for-rhel-8-x86_64-rpms --disable rhel-8-for-x86_64-baseos-rpms
+RUN dnf config-manager --enable openstack-16.2-for-rhel-8-x86_64-rpms --enable rhel-8-for-x86_64-baseos-rpms --enable advanced-virt-for-rhel-8-x86_64-rpms --enable rhceph-4-tools-for-rhel-8-x86_64-rpms --enable fast-datapath-for-rhel-8-x86_64-rpms
+RUN dnf install openstack-nova-common python3-nova python3-novaclient udev fuse -y
+RUN usermod -u 42436 nova && groupmod -g 42436 nova
+RUN usermod -aG disk nova && usermod -a -G libvirt nova && usermod -aG kolla nova
```

Comment:
@pushkar-mahajan You can use single command here.
usermod -a -G disk,libvirt,kolla nova